tests/src/tests/da/api.rs

```diff
@@ -70,12 +75,14 @@ async fn test_get_commitments_from_peers() {
     lone_validator_config.membership = validator.config().membership.clone();
     let lone_validator = Validator::spawn(lone_validator_config).await.unwrap();
 
+    let test_channel_id = setup_test_channel(executor).await;
```

Comment:
could you share what's the reasoning between always setting up a channel like this? It should work indeed but it's not needed, is this to test some specific functionality?